ibis/mapd/tests/test_client.py

```diff
@@ -84,3 +84,39 @@ def test_union_op(alltypes):
     expr = t1.union(t2)
     with pytest.raises(com.UnsupportedOperationError):
         expr.compile()
+
+
+def test_create_table_schema(con):
+    t_name = 'mytable'
+
+    try:
+        # drop the table if it exists
+        con.drop_table(t_name)
+    except Exception:
+        ...
+
+    schema = ibis.schema([
+        ('a', 'float'),
+        ('b', 'double'),
+        ('c', 'int32'),
+        ('d', 'int64'),
+        ('x', 'point'),
+        ('y', 'linestring'),
+        ('z', 'polygon'),
+        ('w', 'multipolygon')
+    ])
+
+    con.create_table(t_name, schema=schema)
+    t = con.table(t_name)
+
+    assert isinstance(t.a, ir.FloatingColumn)
+    assert isinstance(t.b, ir.FloatingColumn)
+    assert isinstance(t.c, ir.IntegerColumn)
+    assert isinstance(t.d, ir.IntegerColumn)
+    assert isinstance(t.x, ir.PointColumn)
+    assert isinstance(t.y, ir.LineStringColumn)
+    assert isinstance(t.z, ir.PolygonColumn)
+    assert isinstance(t.w, ir.MultiPolygonColumn)
+
+    # drop the table
+    con.drop_table(t_name)
```

Comment:
This can still leak the table if any of the previous code fails after the initial drop. You can either create a fixture and use `try:`/`finally:` in the fixture, or use `try:`/`finally:` in the test.